res/css/views/messages/_MLocationBody.scss

```diff
@@ -20,6 +20,10 @@ limitations under the License.
         height: 300px;
 
         border-radius: $timeline-image-border-radius;
+
+        .maplibregl-ctrl-bottom-right {
```

Comment:
Is it possible to do the same on a DOM element we have control over?
We have no guarantee that this class will remain the same in version updates? I tried changing it, and it seems to do the trick, but haven't tested extensively 

```
.mx_MLocationBody_map {
  z-index: 0;
}
```